web/src/pages/FinishedRepetitions/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Helmet } from 'react-helmet-async';
+import { Typography } from '@mui/material';
+
+import { FinishedRepetitionsPageWrapper } from './styled';
+
+export const FinishedRepetitions = (): JSX.Element => {
+  const { t } = useTranslation('translation', { keyPrefix: 'pages.finishedRepetitions' });
```

Comment:
```suggestion
import { CompletedRepetitionsPageWrapper } from './styled';

export const CompletedRepetitions = (): JSX.Element => {
  const { t } = useTranslation('translation', { keyPrefix: 'pages.completedRepetitions' });
```

Change also file name